dhis-2/dhis-services/dhis-service-dxf2/src/main/java/org/hisp/dhis/dxf2/events/event/JdbcEventStore.java

```diff
@@ -1738,23 +1734,22 @@ private String addLastUpdatedFilters( EventSearchParams params, MapSqlParameterS
         return sqlBuilder.toString();
     }
 
-    private String getCategoryOptionSharingForUser( User user, MapSqlParameterSource mapSqlParameterSource )
+    private String getCategoryOptionSharingForUser( User user, EventSearchParams params )
```

Comment:
Seems like EventSearchParams is mostly first in the method signatures in this class. I would keep it that way to be consistent.